src/tips/topics/configue-togg-auto.md

```diff
@@ -0,0 +1,19 @@
+# Updated menu Site Toggling and Automation
+
+Click **Dark Reader** icon.
+
+<img src="/images/tips/reset-settings-icon.png" alt="Dark Reader icon" style="width: 16rem;" loading="lazy" />
+
+On the top menu you will see two updated settings:
+
+<img src="/images/tips/configue-main.png" style="width: 16rem;" loading="lazy" />
+
+Press on the left settings and customize Site Toggling.
```

Comment:
1. Please add subtitle like `### Site Toggling`.

2. "Press" sounds like a keyboard key press. *"Click the Settings button under website URL button."*.